packages/botonic-api/src/bot-dispatcher/local-bot-dispatcher.ts

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios'
+
+import { BotDispatcher, DispatchArgs } from '.'
+
+export class LocalBotDispatcher implements BotDispatcher {
+  bot
+  constructor(bot: any) {
+    this.bot = bot
+  }
+  async dispatch({
+    input,
+    session,
+    lastRoutePath,
+    websocketId,
+  }: DispatchArgs): Promise<void> {
+    const output = await this.bot.app.input({
+      input,
+      session,
+      lastRoutePath,
+    })
+    await this.sendEvents(output, websocketId)
+  }
+  async sendEvents(botOutput: any, websocketId: string): Promise<void> {
+    const { parsedResponse } = botOutput
+    for (const botEvent of parsedResponse) {
+      try {
+        // @ts-ignore
+        await axios.post(`${WEBSOCKET_URL}send/`, {
```

Comment:
WEBSOCKET_URL comes from webpack and it is available at compile time (here the code: https://github.com/hubtype/botonic/pull/1790/files#diff-f95ed1e6da3c4893c78acd2b974be0e6887361ca26db8f3f343d9c59acbe90fbR47). Do you mean to remove it and leave it without slash?

Reply:
Maybe for clarity it would be better to have it like `${WEBSOCKET_URL}/send/` you mean? Then we should also refactor all generated Pulumi's variables in @botonic/pulumi. (we could do it in another task)

Reply:
Yes you're right, I was saying to have it like you say: `${WEBSOCKET_URL}/send/`. With other URLs we have it like this and I think it's clearer.

I was asking about the access to WEBSOCKET_URL because I've seen also that we're accessing to it using `process.env` (like [here](https://github.com/hubtype/botonic/pull/1790/files#diff-0e6f4257f1fe969d6f4c8c2cd39bd4d263a6580f7b22dfe31d250b383feb818bR20)). Should this `process.env` be removed?

Reply:
yes, whereas locally we are using `WEBSOCKET_URL`, when deployed in production, this webpack variable is omitted, because we cannot know which value will have until Pulumi has deployed all the infrastructure in AWS. Then within pulumi's flow, when this variable is directly injected when creating the lambda function to AWS, and when accessing env vars in aws we do it through process.env